Docs/LinuxOsxInstall.md

```diff
@@ -31,3 +20,31 @@ In general, ensure you have SDL2, OpenGL and dotnet 6 or later. Use the `Yafc` e
 
 ### Flathub
 Note that [the version available on Flathub](https://flathub.org/apps/details/com.github.petebuffon.yafc) is not the Community Edition. Its repo can be found at https://github.com/petebuffon/yafc. 
+
+## OSX installation instructions
+Below are the instructions for the arm64 Macs. To use them for Intel Macs, please apply the changes to the folder `Yafc/lib/osx` instead of `Yafc/lib/osx-arm64`.
+- [Install dotnet core (v8.0 or later)](https://dotnet.microsoft.com/download)
+- Download the lua5.2.1 source, copy the .patch file and apply.
+- Modify the `src/Makefile` to include the following two lines. Note that the second line must start with a tab:
+```
+liblua.dylib: $(CORE_O) $(LIB_O)
+        $(CC) -dynamiclib -o $@ $^ $(LIBS)
+```
+- Run `make macosx` in the root directory
+- Run `make -C src liblua.dylib`. This will make `liblia.a` and `liblua.dylib`.  Move them to `Yafc/lib/osx-arm64` with the names `liblua52.a` and `liblua52.dylib`.
+- To get the SDL libraries, [install brew](https://brew.sh/) and then install the packages:
+```
+brew install SDL2
+brew install SDL2_image
+brew install SDL2_ttf
+```
+- Copy the following files from `/opt/homebrew/lib/` to `Yafc/lib/osx-arm64`:
```

Comment:
copy files from `$(brew --prefix)/lib`, not everyone has it at `/opt/homebrew/lib`

even better:

```
$ for LIB in SDL2 SDL2_image SDL2_ttf; do
  brew install "${LIB}"
  cp "$(brew --prefix)/lib/${LIB}" "<yafc repo>/Yafc/lib/osx-arm64/${LIB}"
done
```

Reply:
Thank you! Good point about the location. 
However, the loop introduces more moving parts, which increases the risk of making a mistake.